main.js

```diff
@@ -75,29 +75,29 @@ projects.forEach((project) => {
   const works = document.createElement('works');
   works.innerHTML = `
   <section class="row d-flex justify-content-around">
-    <div class="col-md-4 my-3">
-        <div class="card">
+    
+        <div class="card my-3 py-3">
             <img src="images/${project.featureImage}" alt="Project ${project.id}">
             <div class="card-body">
-                <h5 class="card-title text-center">${project.name}<br> ${project.company}</h5>
-                <div class="d-flex justify-content-around">
-                    <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[0]}</button>
-                    <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[1]}</button>
-                    <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[2]}</button>
-                    <button class="btn btn-secondary btn-outline-success px-1 rounded-3" disabled>${project.tools[3]}</button>
+                <h3 class="card-title text-center">${project.name}<br> ${project.company}</h3>
+                <div class="d-flex gap-1">
+                    <button class="btn btn-secondary btn-outline-success rounded-3" disabled>${project.tools[0]}</button>
+                    <button class="btn btn-secondary btn-outline-success rounded-3" disabled>${project.tools[1]}</button>
+                    <button class="btn btn-secondary btn-outline-success rounded-3" disabled>${project.tools[2]}</button>
```

Comment:
- [ ] Good job adjusting the Desktop details popup very well :clap: :clap: , Please make sure the mobile version looks like the Figma as you know we don't have to put our own design, for now, let's keep sticking to the Figma design.

I suggest bringing the buttons back at the bottom for the mobile version differently from your desktop which is already perfect :+1: 
   - Your Design:
![youBTNS](https://user-images.githubusercontent.com/97834160/209217080-6917917e-1bdc-47cb-9a68-06c9941d1d6c.png)

   
 - Figma design:
 
![figmaBtns](https://user-images.githubusercontent.com/97834160/209217039-b728ebdc-02a5-4dfc-aece-609565c216bd.png)

- [ OPTIONAL] I think your mobile popup menu(the navigation popup) is set up to display by default please set it to display none by default so that it appears only when the user clicks on the hamburger button
![mob](https://user-images.githubusercontent.com/97834160/209217514-f16d9f80-e73d-4791-878e-6c4f98daa4a9.png)